packages/graphql-language-service-server/src/GraphQLLanguageService.ts

```diff
@@ -24,29 +24,37 @@ import {
   Outline,
   OutlineTree,
   GraphQLCache,
-} from 'graphql-language-service-types';
+} from 'graphql-language-service';
 
 import { GraphQLConfig, GraphQLProjectConfig } from 'graphql-config';
+
 import {
   Hover,
   SymbolInformation,
   SymbolKind,
 } from 'vscode-languageserver-types';
 
 import { Kind, parse, print } from 'graphql';
-import { getAutocompleteSuggestions } from './getAutocompleteSuggestions';
-import { getHoverInformation, HoverConfig } from './getHoverInformation';
-import { validateQuery, getRange, DIAGNOSTIC_SEVERITY } from './getDiagnostics';
+import { getAutocompleteSuggestions } from 'graphql-language-service/src/interface/getAutocompleteSuggestions';
+import {
+  getHoverInformation,
+  HoverConfig,
+} from 'graphql-language-service/src/interface/getHoverInformation';
+import {
+  validateQuery,
+  getRange,
+  DIAGNOSTIC_SEVERITY,
+} from 'graphql-language-service/src/interface/getDiagnostics';
 import {
   getDefinitionQueryResultForFragmentSpread,
   getDefinitionQueryResultForDefinitionNode,
   getDefinitionQueryResultForNamedType,
   DefinitionQueryResult,
-} from './getDefinition';
+} from 'graphql-language-service/src/interface/getDefinition';
```

Comment:
reaching into the src dir for another module feels quite strange, wouldn't these be dist files?

Reply:
Ah good catch! Spent so much time cleaning up the autorename bugs